packages/expo-media-library/ios/MediaLibraryExceptions.swift

```diff
@@ -0,0 +1,145 @@
+import ExpoModulesCore
+
+internal class MethodUnavailableException: Exception {
+  override var reason: String {
+    "presentLimitedLibraryPickerAsync is only available on iOS >= 15"
+  }
+}
+
+internal class MediaLibraryPermissionsException: Exception {
+  override var reason: String {
+    "Media Library permission is required to do this operation"
+  }
+}
+
+internal class FileExtensionException: Exception {
+  override var reason: String {
+    "Could not get the file's extension"
+  }
+}
+
+internal class UnsupportedAssetTypeException: Exception {
+  override var reason: String {
+    "This file type is not supported yet"
+  }
+}
+
+internal class UnreadableAssetException: GenericException<String> {
+  override var reason: String {
+    "File \(param) isn't readable"
+  }
+}
+
+internal class SaveAssetException: Exception {
+  override var reason: String {
+    "Asset couldn't be saved to photo library"
+  }
+}
+
+internal class MissingPListKeyException: GenericException<String> {
+  override var reason: String {
+    "This app is missing \(param). Add this entry to your bundle's Info.plist."
+  }
+}
+
+internal class MissingFileException: GenericException<String> {
+  override var reason: String {
+    "Couldn't open file: \(param). Make sure if this file exists."
+  }
+}
+
+internal class SaveVideoException: Exception {
+  override var reason: String {
+    "This video couldn't be saved to the Camera Roll album"
+  }
+}
+
+internal class SaveAlbumException: Exception {
+  override var reason: String {
+    "Couldn\'t add assets to album"
```

Comment:
```suggestion
    "Couldn't add assets to album"
```
I think it's not necessary to escape the `'` character in `"`.